### PR TITLE
[Doc] Add *InfluxDB sink connector guide*

### PR DESCRIPTION
**Pull request by Anonymitaet**
Fix https://github.com/apache/pulsar/issues/5015

### CODE REVIEW

**Comment by Anonymitaet** [comment]
@tuteng could you please help review? Thank you

File: site2/docs/io-influxdb-sink.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+The configuration of the InfluxDB sink connector has the following properties.
+
+### Property
+
+| Name | Type|Required | Default | Description 
+|------|----------|----------|---------|-------------|
+| `influxdbUrl` |String| true|" " (empty string) | The URL of the InfluxDB instance. |
+| `username` | String|false| " " (empty string) |The username used to authenticate to InfluxDB. |
+| `password` | String| true|" " (empty string)  | The password used to authenticate to InfluxDB. |
```
Comment (tuteng) [inline site2/docs/io-influxdb-sink.md]: The field `password` is optional

Comment (Anonymitaet) [reply site2/docs/io-influxdb-sink.md]: Thank you, I've corrected. Could you PTAL?

**Comment by tuteng** [comment]
run integration tests
run java8 tests

**Comment by Anonymitaet** [comment]
run integration tests
run java8 tests

**Comment by Anonymitaet** [comment]
run integration tests

**Pull request merged**